packages/suite-desktop/package.json

```diff
@@ -39,28 +39,47 @@
         },
         "files": [
             "build/**/*",
-            "!build/static/bin/**/*",
+            "!build/**/*.js.map",
             "dist/**/*.js",
-            "!dist/__**",
             "package.json",
-            "!**/node_modules/@trezor/**/src",
-            "!**/node_modules/@trezor/**/coverage",
-            "!**/node_modules/@trezor/**/build",
-            "!**/node_modules/@trezor/**/scripts",
-            "!**/node_modules/@trezor/**/webpack",
-            "!**/node_modules/@trezor/**/integration",
-            "!**/node_modules/@trezor/**/*.bin",
+            "!build/static/**/{favicon,icons,bin,browsers,suite-web-landing}",
+            "!bin/firmware",
+            "!node_modules/@suite-common",
+            "!node_modules/@trezor/**/{src,coverage,build,scripts,webpack,integration,e2e,libDev}",
```

Comment:
I am kind of worrying about maintaining this list, but it could be probably removed when we will actually remove build:lib and whole electron layer will be bundled (except some core electron deps). I also thinked about what @szymon said about duplicating some dependencies, and I agree that it can happen, but I think there will be only few of them and from experiments I did with that it looks that thanks to bundling a minification result package size will be even smaller than it is now.